include/xtensor/xchunked_array.hpp

```diff
@@ -427,8 +427,8 @@ namespace xt
     template <class Idx>
     inline std::pair<size_t, size_t> xchunked_array<CS, EX>::get_chunk_indexes_in_dimension(size_t dim, Idx idx) const
     {
-        size_t index_of_chunk = idx / m_chunk_shape[dim];
-        size_t index_in_chunk = idx - index_of_chunk * m_chunk_shape[dim];
+        size_t index_of_chunk = static_cast<size_t>(idx) / m_chunk_shape[dim];
+        size_t index_in_chunk = static_cast<size_t>(idx) - index_of_chunk * m_chunk_shape[dim];
```

Comment:
```suggestion
        size_t index_of_chunk = static_cast<std::size_t>(idx) / m_chunk_shape[dim];
        size_t index_in_chunk = static_cast<std::size_t>(idx) - index_of_chunk * m_chunk_shape[dim];
```

Reply:
Theoretically, yes, but the file use `size_t` without `std` namespace everywhere else. let's keep as is for now (for homgeneity) and make a pass to replace them all later.